### ISSUE TITLE
[BUG] TOGA always displayed

### ISSUE THREAD

**Issue by phildeg31** [flybywiresim/a32nx#2017]
**Mod Version**
developer verion

**Describe the bug**
TOGA is always displayed above the pourcentage

**To Reproduce**
First, the throttle is on iddle position but a poucentage is displayed instead of 0% and I have to move the throttle to have a good poucentage.
Then, when I move the throttle, only TOGA is displayed.

**Expected behavior**
0% when throttle is on iddle position.
CLB, MCD and iddle text above the pourcentage according to throttle position and not only TOGA
Last stable version (0.4.1) worked fine for this but last developer version has this issue.

**Actual behavior**
Only TOGA

**References**
https://zupimages.net/viewer.php?id=20/47/4gfx.jpg

**Comment by MisterChocker** [comment]
This is called thrust rating modes and correct behavior.
![image](https://user-images.githubusercontent.com/1857322/99818603-b770d280-2b4e-11eb-80fa-08c3720e2f84.png)

**Comment by sharma** [comment]
It shouldn't always be displaying TOGA regardless of the actual thrust rating though. Even when I have the throttle set to CLB it displays TOGA. This wasn't the case in previous development versions.

**Comment by MisterChocker** [comment]
Yes, could you please give me more information on when TOGA is shown, situations, flight phases etc.

**Comment by martysweet** [comment]
I found this was an issue on build:
```
built: 2020-11-10T15:19:38+00:00
ref: refs/heads/master
sha: 6cb5116649c223cba11e7eec6ee39fcba9ecfb68
actor: Benjozork
event_name: push
```

However, it seems to be fixed/I didn't notice it on the latest:
`{ "built": "2020-11-20T22:55:09+00:00", "ref": "refs/heads/master", "sha": "2665e17d24ac35a2b72bdc005ef1d9df087d6ef9", "actor": "wpine215", "event_name": "push" }`

@phildeg31 Whats the contents of your build_info?

**Comment by phildeg31** [comment]
My build info:
{ "built": "2020-11-19T14:59:45+00:00", "ref": "refs/heads/master", "sha": "5e57e8dd9fbcf3cbbefd119606df2576da85bd7e", "actor": "Benjozork", "event_name": "push" }

I will try with the last one.

**Comment by bisinchi** [comment]
I tried today with a master downloaded yesterday and also found the issue to still be present. if I had throttles set to CLB or MCT detent, TOGA would still be displayed on top right of the ECAM display.

**Comment by MisterChocker** [comment]
> I tried today with a master downloaded yesterday and also found the issue to still be present. if I had throttles set to CLB or MCT detent, TOGA would still be displayed on top right of the ECAM display.

Please explain situation? I assume you were on ground? If so you had not set a flex temp.

**Comment by bisinchi** [comment]
![320NEO](https://user-images.githubusercontent.com/71987677/100200068-653d0200-2efe-11eb-9052-ded002449514.jpg)
No flex temp entered. 
Just loaded the airplane on MSFS and took off for circuits.
And also had Toga displayed after going to CLB detent. Then problem disappeared on my test today.

**Comment by MisterChocker** [comment]
Does this still occur on latest master?

**Comment by bisinchi** [comment]
Without inserting flighplan, nor speed, I have TOGA indication on ground even with throttles below CLB.
After take off if I set the throttles to MCT, then TOGA indication changes to MCT and when I change to CLB indication also changes to CLB. But If I change from TOGA straight to CLB, indication still says TOGA. Only way to have it changed is by using MCT detent. This is with latest Dev branch downloaded this morning.

**Comment by MisterChocker** [comment]
> Without inserting flighplan, nor speed, I have TOGA indication on ground even with throttles below CLB.

Intended behavior.



> If I change from TOGA straight to CLB, indication still says TOGA

How do you go from TOGA directly to CLB?

**Comment by bisinchi** [comment]
I move the throttle quickly from TOGA to CLB without stopping at MCT.

**Comment by MisterChocker** [comment]
Closing since no response

**Comment by bisinchi** [comment]
I did reply 9 days ago.
    On Saturday, January 9, 2021, 08:13:25 PM GMT+1, DerL30N <notifications@github.com> wrote:  
 
 


Closing since no response

**Comment by bisinchi** [comment]
Hi,
Just tried again today, seem to have same behavior, but bear in mind that this is happening when taking off without inserting speeds, weights, anything on the FMC. Not what you are supposed to do anyway, so I wouldn't spend time in that issue right now, maybe to be looked at later as an improvement.
 
    On Sunday, January 10, 2021, 11:03:42 AM GMT+1, Francois de la Simone <bisinchi@yahoo.co.uk> wrote:  
 
  I did reply 9 days ago.
    On Saturday, January 9, 2021, 08:13:25 PM GMT+1, DerL30N <notifications@github.com> wrote:  
 
 


Closing since no response
